.github/workflows/build-ffmpeg.yml

```diff
@@ -67,7 +67,7 @@ jobs:
               --enable-encoder=h264_videotoolbox,hevc_videotoolbox
               --enable-videotoolbox
           - os_type: macos
-            os: macos-12
+            os: macos-11
```

Comment:
I'd imagine the videotoolbox api is pretty stable... I'd lean towards keeping this as is (macOS 13 is actually the latest). Additionally, all arm macs have no compatibility issues with the latest OS and mac users tend to upgrade as soon as a new one is released.

Reply:
It seems that the way to avoid compatibility issues is to build on the oldest version possible. If it doesn't hurt anything I'd prefer to move this to 11 for now (x86_64 is already on macos-11... they should certainly match).

This is a reference from PyInstaller: https://pyinstaller.org/en/stable/usage.html?highlight=oldest#macos (I know this isn't a python library but I think we can still avoid problems like I'm currently facing in Ubuntu builds by following this practice).